### PR TITLE
Clean up summary logs

### PR DESCRIPTION
**Pull request by ewels**
It struck me that the summary log is getting pretty long, and that there is quite a lot of stuff there that is either not interesting or pretty rare flags. I did a bit of housecleaning.:

Before:
image

After:
image

I also moved some params initialisations into the `nextflow.config` file

PR will be easier to look at with whitespace changes ignored 👀

### CODE REVIEW

**Comment by ewels** [comment]
Also snuck in responses to @drpatelh's comments on #133

File: main.nf [hunk 1]
```diff
@@ -500,26 +483,27 @@ if( params.skip_trimming ){
...
+        tpc_r1 = three_prime_clip_r1 > 0 ? "--three_prime_clip_r1 $three_prime_clip_r1" : ''
+        tpc_r2 = three_prime_clip_r2 > 0 ? "--three_prime_clip_r2 $three_prime_clip_r2" : ''
         rrbs = params.rrbs ? "--rrbs" : ''
-        multicore = ''
-        if( task.cpus ){
-            ccore = (((task.cpus as int) - 3) / 3) as int
-            if( ccore > 1 ){
-              multicore = "--cores $ccore"
-            }
+        cores = 1
```
Comment (drpatelh) [inline main.nf]:
```suggestion
        def cores = 1
```

Comment (drpatelh) [reply main.nf]: I think this is one instance where I know `def` should be used :laughing:

Comment (ewels) [reply main.nf]: Why?

Comment (drpatelh) [reply main.nf]: Initialising the type of a variable I believe before you give it another value. Probably dont have to but I always do. Let me see if I can dig something up.

Comment (drpatelh) [reply main.nf]: http://grails.asia/groovy-def-declaration-examples

Comment (drpatelh) [reply main.nf]: I dont know if its going to have much difference to be honest. Ive just got into the habit of using it and as far as I know its a halfway house to using the correct Groovy implementation in Nextflow.
e.g.
https://github.com/drpatelh/nanoseq/blob/50b2e779d4592739682af658a17423876b2abc07/main.nf#L256-L270

Comment (phue) [reply main.nf]: There are two other instances of a multicore logic in the pipeline. Maybe they should be adjusted to match this implementation?

https://github.com/nf-core/methylseq/blob/0cae7597d70cdfc41178a3e6428a896be9421c55/main.nf#L571

https://github.com/nf-core/methylseq/blob/0cae7597d70cdfc41178a3e6428a896be9421c55/main.nf#L672

Comment (ewels) [reply main.nf]: Yes, probably - but can we add an issue for this and do it another time? It shouldn't affect the way that the pipeline runs I think and I am keen to get this minor / patch release ASAP (and tomorrow is my last day before christmas holidays 🙄)

**Comment by ewels** [comment]
I'm going to be a bit cheeky and merge this if there aren't any major issues with the code here. It will roll in to the already open PR https://github.com/nf-core/methylseq/pull/133 so we can continue discussion there.

Travis is taking hours to run at the moment so I'm keen to let it sit in a queue overnight if possible instead of during valuable daylight hours 😎

**Pull request merged**